src/Network/NWPathMonitor.cs

```diff
@@ -72,7 +86,7 @@ static void TrampolineUpdatedSnapshot (IntPtr block, IntPtr path)
 		static extern unsafe void nw_path_monitor_set_update_handler (IntPtr handle, void *callback);
 
 		[BindingImpl (BindingImplOptions.Optimizable)]
-		public void SetUpdatedSnapshotHandler (Action<NWPath> callback)
+		void InternalSetUpdatedSnapshotHandler (Action<NWPath> callback)
```

Comment:
Minor but I think we usually go `_SetUpdatedSnapshotHandler` for internal stuff.